spec/lib/common/models/concerns/cache_aside_spec.rb

```diff
@@ -52,7 +52,7 @@
         allow(VAProfile::Models::V3::Person).to receive(:build_from).and_return(person)
       end
 
-      describe '#do_cached_with', :initiate_vaprofile, :skip_vet360 do
+      describe '#do_cached_with', :skip_vet360 do
```

Comment:
Where are these tags defined and what are they doing? I'm assuming it must control how the test is executed or some kind of setup/config but I'm having a hard time finding where they're defined

Reply:
My team is going to start adding tags to our tests.  See  [this document](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/tutorials/testing/rspec_tags.md) for details.  

@LindseySaari I figured I'd mention it here because of your comment.

Reply:
Thanks @tpharrison!

Reply:
@LindseySaari initiate_vaprofile will build a user with the new contact information service attributes, [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/va_profile/stub_vaprofile_user.rb).